qiskit/extensions/quantum_initializer/isometry.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty):
 
         self.num_ancillas_zero = num_ancillas_zero
         self.num_ancillas_dirty = num_ancillas_dirty
+        self.eps = eps
```

Comment:
should probably be `self._eps` (private) so you can't change it after making an Isometry instruction.